apps/src/lib/kits/maker/boards/microBit/MicroBitConstants.js

```diff
@@ -34,6 +34,12 @@ export const MB_SENSOR_VARS = [
   'compass'
 ];
 
+// milliseconds between samples for sensors
+export const SAMPLE_RATE = 50;
```

Comment:
"Sample rate" here would be how many times we sample per second (i.e. 1000 / 50 = 20).  What we appear to have here is actually SAMPLE_INTERVAL.